presto-accumulo/pom.xml

```diff
@@ -323,5 +332,11 @@
             <artifactId>javax.annotation-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>2.5.0</version>
```

Comment:
I'm assuming this is added here because `accumulo-minicluster` needs it. In that case, we should remove the exclusion there, rather than manually adding it here (and hoping the versions don't get out of sync).